docs/webui.md

```diff
@@ -0,0 +1,206 @@
+
+|Key            |Description      |
+|---------------|-----------------|
+|catalog-backend|`hive`, or `jdbc`|
+
+  - `hive`
+
+  ![props-iceberg-hive](assets/webui/props-iceberg-hive.png)
+
+  |Key      |Description                     |
+  |---------|--------------------------------|
+  |uri      |Iceberg catalog uri config      |
+  |warehouse|Iceberg catalog warehouse config|
+
+  - `jdbc`
+
+  ![props-iceberg-jdbc](assets/webui/props-iceberg-jdbc.png)
+
+  |Key          |Description                                                                                            |
+  |-------------|-------------------------------------------------------------------------------------------------------|
+  |uri          |Iceberg catalog uri config                                                                             |
+  |warehouse    |Iceberg catalog warehouse config                                                                       |
+  |jdbc-driver  |"com.mysql.jdbc.Driver" or "com.mysql.cj.jdbc.Driver" for MySQL, "org.postgresql.Driver" for PostgreSQL|
+  |jdbc-user    |jdbc username                                                                                          |
+  |jdbc-password|jdbc password                                                                                          |
+
+- mysql
+
+Following: [JDBC MySQL catalog](jdbc-mysql-catalog)
+
+![props-mysql](assets/webui/props-mysql.png)
+
+|Key          |Description                                                                                        |
+|-------------|---------------------------------------------------------------------------------------------------|
+|jdbc-driver  |JDBC URL for connecting to the database. e.g. `com.mysql.jdbc.Driver` or `com.mysql.cj.jdbc.Driver`|
+|jdbc-url     |e.g. `jdbc:mysql://localhost:3306`                                                                 |
+|jdbc-user    |The JDBC user name                                                                                 |
+|jdbc-password|The JDBC password                                                                                  |
+
+- postgresql
+
+![props-pg](assets/webui/props-pg.png)
+
+|Key          |Description                                          |
+|-------------|-----------------------------------------------------|
+|jdbc-driver  |e.g. `e.g. org.postgresql.Driver`                    |
+|jdbc-url     |e.g. `jdbc:postgresql://localhost:5432/your_database`|
+|jdbc-user    |The JDBC user name                                   |
+|jdbc-password|The JDBC password                                    |
+|jdbc-database|e.g. `pg_database`                                   |
+
+After vertified inputing these fields, clicking on the `CREATE` button to creates a catalog.
+
+![created-catalog](assets/webui/created-catalog.png)
+
+Clicking the icon button - `←(left arrow)` will redirect to metalake page.
+
+### Schema
+
+Under Construction...
```

Comment:
You'd better have a basic list view and detailed view for schema and table.

Reply:
> You'd better have a basic list view and detailed view for schema and table.

Is the following format acceptable?
<img width="407" alt="image" src="https://github.com/datastrato/gravitino/assets/17310559/a4905a67-6514-47f1-89d7-8be43ad6800c">

Reply:
Yes, it would be better you also have screenshots to show the schemas and tables.